zfit/models/dist_tfp.py

```diff
@@ -933,3 +933,105 @@ class BifurGaussPDFRepr(BasePDFRepr):
+        name: str = "Gamma",
+    ):
+        """Gamma distribution.
+
+        The gamma shape is parametrized here with `gamma`, `beta` and `mu`.
+        We follow the same parametrization `as RooFit <https://root.cern.ch/doc/master/classRooGamma.html>`_.
+        The gamma shape is defined as
+
+        .. math::
+
+            f(x \\mid \\gamma, \\beta, \\mu) = (x - \\mu)^{\\gamma - 1} \\exp{\\left(-\\frac{x - \\mu}{\\beta}\\right)} / Z
+
+        with the normalization over [0, inf] of
+
+        .. math::
+
+            Z = \\Gamma(\\gamma) \\beta^{\\gamma}
+
+        The normalization changes for different normalization ranges
+
+        Args:
+            gamma: Shape parameter of the gamma distribution
+            beta: Scale parameter of the gamma distribution
+            mu: Shift of the distribution
+            obs: |@doc:model.init.obs| Observables of the
+               model. This will be used as the default space of the PDF and,
+               if not given explicitly, as the normalization range.
+               If the observables are binned and the model is unbinned, the
+               model will be a binned model, by wrapping the model in a
+               :py:class:`~zfit.pdf.BinnedFromUnbinnedPDF`, equivalent to
+               calling :py:meth:`~zfit.pdf.BasePDF.to_binned`.
+
+               The default space is used for example in the sample method: if no
+               sampling limits are given, the default space is used.
+
+               The observables are not equal to the domain as it does not restrict or
+               truncate the model outside this range. |@docend:model.init.obs|
+            extended: |@doc:pdf.init.extended| The overall yield of the PDF.
+               If this is parameter-like, it will be used as the yield,
+               the expected number of events, and the PDF will be extended.
+               An extended PDF has additional functionality, such as the
+               ``ext_*`` methods and the ``counts`` (for binned PDFs). |@docend:pdf.init.extended|
+            norm: |@doc:pdf.init.norm| Normalization of the PDF.
+               By default, this is the same as the default space of the PDF. |@docend:pdf.init.norm|
+            name: |@doc:model.init.name| Human-readable name
+               or label of
+               the PDF for better identification. |@docend:model.init.name|
+        """
+        gamma, beta, mu = self._check_input_params_tfp(gamma, beta, mu)
+        params = {"gamma": gamma, "beta": beta, "mu": mu}
+
+        def dist_params():
+            return {"concentration": gamma.value(), "rate": 1 / beta.value(), "loc": mu.value()}
+
+        distribution = tfp.distributions.Gamma
+        super().__init__(
+            distribution=distribution,
+            dist_params=dist_params,
+            obs=obs,
+            params=params,
+            name=name,
+            extended=extended,
+            norm=norm,
+        )
+
+    @property
+    def distribution(self):
```

Comment:
I would suggest to refactor this and remove the custom `distribution` property: above, `distribution` can be any callable that takes the parameters and returns a distribution, so we can just define this above in the init, approx
```
def distribution(mu, sigma,...):
    return TransformedDist(......)
```

(reduces duplicated code and streamlines the pdf)

Reply:
Please take a look and let me know if this is what you were aiming for